.github/workflows/rotki_nightly.yml

```diff
@@ -0,0 +1,189 @@
+name: Rotki Nightly
```

Comment:
So we will need to keep this in sync with any changes that happen to the rotki_ci.yml workflow?

Reply:
Unfortunately yes :'(